proto/slinky/mm2/v1/params.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package slinky.mm2.v1;
+
+option go_package = "github.com/skip-mev/slinky/x/mm2/types";
+
+// Params defines the parameters for the x/marketmap module.
+message Params {
+  // MarketAuthorities is the list of authority accounts that are able to
+  // control updating the marketmap.
+  repeated string market_authorities = 1;
```

Comment:
Maybe we can do this in a followup:

I thought our idea was to have a 2 tiered permissions system in which participants could update the mm, and a single "admin" authority could add/remove participants.

Reply:
yeah - I added this in a follow up https://github.com/skip-mev/slinky/pull/293